packages/mineral-ui-tokens/scripts/build-tokens.js

```diff
@@ -147,3 +164,16 @@ configurations.forEach(({ transform, format, fileName }) => {
     })
     .catch((error) => error && console.log('Error converting tokens: ', error));
 });
+
+websiteConfigurations.forEach(({ transform, format, fileName }) => {
```

Comment:
Rather than having two `forEach`s with nearly identical calls to `theo`, you could have an optional property in the configuration objects, e.g. `writeDir`, that defaults to `SRC_DIR` and then use it in the `path.join`.